guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.guvnor.structure.repositories;
+
+public interface PullRequest {
+
```

Comment:
@Salaboy  I tend to agree on the missing author bit. But this information can be obtained from the source repository owner. I think this will be refined on the next phase of this work... that is create the PR UXD.